src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+                +"____________________________________________________________\n";
+        System.out.println(logo);
+        Scanner stringScanner = new Scanner(System.in);
+        String userQuery = stringScanner.nextLine();
+        String userQueryReturn = "";
+
+        while(!userQuery.equalsIgnoreCase("bye")){
+            if(userQuery.contains("done")) {
+                int doneIndex = Integer.parseInt(userQuery.substring(5));
+                taskItems.get(doneIndex - 1).setStatus(true);
+                userQueryReturn = "____________________________________________________________\n"
+                        + " Nice! I've marked this task as done: \n"
+                        + " [" + taskItems.get(doneIndex - 1).getTypeIcon() + "][" + taskItems.get(doneIndex - 1).getStatusIcon() + "] " + taskItems.get(doneIndex - 1).getDescription() + "\n"
+                        + "____________________________________________________________\n";
+                System.out.println(userQueryReturn);
+                saveToFile(taskItems);
+                userQuery = stringScanner.nextLine();
+            }else if(userQuery.contains("delete")){
+                int doneIndex = Integer.parseInt(userQuery.substring(7));
+                Task removedItem = taskItems.remove(doneIndex);
+                userQueryReturn = "____________________________________________________________\n"
+                        + " Noted. I've removed this task: \n"
+                        + "   [" + removedItem.getTypeIcon() + "][" + removedItem.getStatusIcon() + "] " + removedItem.getDescription() + "\n"
+                        + " Now you have "+ taskItems.size() + " tasks in the list.\n"
+                        + "____________________________________________________________\n";
+                System.out.println(userQueryReturn);
+                saveToFile(taskItems);
+                userQuery = stringScanner.nextLine();
+            }else if(userQuery.contains("todo")||userQuery.contains("deadline")||userQuery.contains("event")) {
+                taskItems.add(new Task(userQuery));
+                itemCount++;
+                if(userQuery.equalsIgnoreCase("todo") || userQuery.equalsIgnoreCase("deadline") || userQuery.equalsIgnoreCase("event")){
+                    userQueryReturn = "____________________________________________________________\n"
+                            + taskItems.get(taskItems.size() - 1).getDescription()
+                            + "\n____________________________________________________________\n";
+                }else {
+                    userQueryReturn = "____________________________________________________________\n"
+                            + " Got it. I've added this task: \n"
+                            + "   [" + taskItems.get(taskItems.size() - 1).getTypeIcon() + "][" + taskItems.get(taskItems.size() - 1).getStatusIcon() + "] " + taskItems.get(taskItems.size() - 1).getDescription() + "\n"
```

Comment:
Consider breaking this line since it's a little too long (it's not showing up on GitHub towards the end)